src/components/Chat/MessageForm/MessageForm.js

```diff
@@ -23,8 +23,12 @@ function MessageForm() {
   const dispatch = useDispatch();
 
   return (
-    <form className="MessageForm" onSubmit={(event) => handleSubmit(event, dispatch, textInput)}>
-      <input type="text" id="text" ref={textInput} />
+    <form
+      data-testid="chatbox"
```

Comment:
I'd like to talk about test, [`getByTestId`](https://testing-library.com/docs/dom-testing-library/api-queries#bytestid), and this `data-testid` attribute too, please.